Resources/Prototypes/Reagents/narcotics.yml

```
@@ -120,7 +120,7 @@
   meltingPoint: 170.0
   metabolisms:
     Narcotic:
-      metabolismRate: 1.0
+      metabolismRate: .5   #Harmony Change
```

Small nitpick but I prefer having a leading zero
```suggestion
      metabolismRate: 0.5   #Harmony Change
```